apps/web/pages/[user]/[type].tsx

```diff
@@ -279,16 +278,20 @@ async function getDynamicGroupPageProps(context: GetServerSidePropsContext) {
     },
   });
 
+  const users = usersWithoutAvatar.map((user) => ({
+    ...user,
+    avatar: `${WEBAPP_URL}/${user.username}/avatar.png`,
+  }));
+
   if (!users.length) {
     return {
       notFound: true,
     };
   }
 
   // sort and be in the same order as usernameList so first user is the first user in the list
-  let sortedUsers: typeof users = [];
   if (users.length > 1) {
-    sortedUsers = users.sort((a, b) => {
```

Comment:
`sortedUsers` is redundant, as `users` is mutated anyway.
see [apps/web/pages/[user].tsx](https://github.com/calcom/cal.com/blob/main/apps/web/pages/%5Buser%5D.tsx#L308C5-L308C5)